usr/sbin/p11sak/p11sak.c

```diff
@@ -978,6 +1049,14 @@ static CK_RV tok_key_list_init(CK_SESSION_HANDLE session, p11sak_kt kt,
                     p11_get_ckr(rc));
             return rc;
         }
+    } else if (kt == kt_ALL) {
+        rc = funcs->C_FindObjectsInit(session, NULL, 0);
+        if (rc != CKR_OK) {
+            printf("C_FindObjectInit failed in tok_key_list_init() (error code 0x%lX: %s)\n", rc,
```

Comment:
Error messages normally go to stderr.  Is the output to stdout desired?

Reply:
There seems to be a mixture of error messages going to stdout or stderr in the p11sak code.... While you are at it, please change ALL error messages to go to stderr.

Reply:
Done in new commit